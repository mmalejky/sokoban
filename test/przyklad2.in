@-------
----b++-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------

b6
b6
b6
b6
b8
b8
b4
a2
0
0
0
0
0
b4
.
