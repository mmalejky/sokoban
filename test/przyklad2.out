@-------
----b++-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
----@B+-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
-----*B-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
-----+*b
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
-----+*b
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
-------b
-----++@
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
-------b
-----++@
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
-------b
-----++@
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
-------b
-----++@
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
-----+*b
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
-----*B-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
----@B+-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
@-------
----b++-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
@-------
----b++-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
--------
---b@++-
--####--
--#--#--
--#-a#--
--#--#--
--####--
--------
--------
